index.js

```diff
@@ -97,18 +99,20 @@ const validateFlags = (flags, options) => {
 	}
 };
 
-const meow = (helpText, options) => {
+const meow = (helpText, options = {}) => {
 	if (typeof helpText !== 'string') {
 		options = helpText;
 		helpText = '';
 	}
 
-	const foundPackage = readPackageUpSync({
+	const foundPackage = readPackageSync({
+		cwd: options.packagePath ? dirname(fileURLToPath(options.packagePath)) : process.cwd(),
 		normalize: false
 	});
 
 	options = {
-		pkg: foundPackage ? foundPackage.packageJson : {},
+		packagePath: process.cwd(),
```

Comment:
The default value is `process.cwd()`, but it's useless. Do we need to set it as a required option?